docs/medical-api/more-info/api-tools.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Api Tools"
```

Comment:
thinking about this some more, should we have 3 separate pages for:

- SDKs
- OpenAPI spec
- Postman collection

"Api Tools" is very vague, and hard to know we even have these available at the surface